settings.gradle

```diff
@@ -1,2 +1,2 @@
-include ':'
+include ':owncloudComLib'
```

Comment:
I would keep the name `android-library` and also the middle packages `com.owncloud.android` instead of directly `lib`

Reply:
Thinking about there is an other unknow app using this lib next to other libs (not Owncloud itself), and any included lib is called `android-library`
I have it in a current project, the libs are called `sdk` and 'library', ... -sad-
With `android` you know an obsolete information
With `library` you know it's an library  ...congratulations

Reply:
> Thinking about there is an other unknow app using this lib next to other libs (not Owncloud itself), and any included lib is called `android-library`
> I have it in a current project, the libs are called `sdk` and 'library', ... -sad-
> With `android` you know an obsolete information
> With `library` you know it's an library ...congratulations

with the current `owncloud-android-library` you know that this is an android based library to be used in ownCloud, you have all the information and do not need any other information. This is my HO. I do not see any reason to switch the modules' names.

Reply:
ok, I will apply `owncloud-android-library` but keep in mind, based on the name you don't know does is deliver GUI elements, communication stuff, general helpers or what else

Reply:
I still do not get what additional information the previous name you had - owncloudComLib - provides in comparison with `owncloud-android-library`. Com means communication? If so, you could have detailed it before, I understood it as a domain

Reply:
@davigonz anyway, names are the most complex issue in software development. I followed the suggestion from @jesmrec